model/docs/development/ADRs/0001_model_physics_components_interface.md

```diff
@@ -33,14 +34,12 @@ the user to decide which update should be applied to the model state prior to th
 
 **Concerns**
 
-- If a component depends on the tendencies produced by another component, it is unclear whether the effect of this other component 
-can be cleanly isolated.
-
-
+- If a component depends on the tendencies produced by another component, it is unclear whether the effect of this other component
```

Comment:
```suggestion
- If a component depends on another component's tendencies, it is unclear whether the effect of the latter
```

Reply:
ADRs are internal technical documentation so I'd rather not modify its content, specially if written by other developers, in an unrelated PR fixing tooling configuration.